corehq/apps/fixtures/tests/test_upload.py

```diff
@@ -684,6 +684,26 @@ def error_tx():
             self.upload([(None, 'N', 'orange')])
         clear_cache.assert_called_once()
 
+    def test_row_added_clears_its_table_cache(self):
+        self.upload([(None, 'N', 'apple')])
+        table = self.get_table()
+        apple_id = self.get_rows(None)[0].id.hex
+
+        with patch.object(mod, "clear_fixture_cache") as mock_clear:
+            self.upload([(apple_id, 'N', 'apple'), (None, 'N', 'banana')])
+            self.assertEqual(len(mock_clear.call_args[0][1]), 1)
+            self.assertIn(table.id, mock_clear.call_args[0][1])
```

Comment:
nit: I think reducing this to a single assertion will make test failures easier to debug if these assertions ever fail. This same idea can be applied to each of the new tests in this module.
```suggestion
            self.assertEqual(mock_clear.call_args[0][1], [table.id])
```